.dockerignore

```
@@ -1,5 +1,6 @@
 target
 !target/debug/driver
+!target/orderbook.bin
```

Maybe something like 
```suggestion
!target/orderbook*.bin
```

To allow for different orderbooks on mainnet and rinkeby?